ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-release-1.24.yaml

```diff
@@ -241,7 +241,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          pilot 'TestGatewayConformance|TestGateway/managed-owner|TestCNIVersionSkew|TestGateway|TestAuthZCheck|TestKubeInject|TestRevisionTags|TestUninstallByRevision|TestUninstallWithSetFlag|TestUninstallCustomFile|TestUninstallPurge'"
+          pilot 'TestGatewayConformance|TestGateway/managed-owner|TestCNIVersionSkew|TestGateway'"
```

Comment:
Waiting for https://github.com/openshift-service-mesh/istio/pull/355 to be merged

Reply:
It's now merged